index.html

```diff
@@ -14,18 +14,23 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">My Logo</a>
+            <ul class="navbar-list">
 
-            <button class="menu-button">
+            <li class="menu-icon">
                 <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </button>
+            </li>
+            <li class="desktop-nav-list">Portfolio</li>
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
 
         </nav>
 
 
     </header>
 
     <section class="container">
```

Comment:
- [ ] I will suggest you make sure that the background-image of the headline matches the figma design 👍🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213546121-7eabe7b2-12ed-47da-9be5-82b96ca4446b.png)

FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213546236-8f2654e1-6689-4a47-99d7-95fa847f612c.png)